lib/rubocop/cop/rspec/context_description.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # `context` block descriptions should start with 'when'.
+      #
+      # @example
+      #   # bad
+      #   context 'the display name not present' do
+      #     ...
+      #   end
+      #
+      #   # good
+      #   context 'when the display name is not present' do
+      #     ...
+      #   end
+      class ContextDescription < Cop
+        MSG = 'Start context description with `when`.'
```

Comment:
I believe this is a false offense considering a magic `frozen_string_literal` comment above.

```
lib/rubocop/cop/rspec/context_description.rb:19:15: C: Style/MutableConstant: Freeze mutable objects assigned to constants.
        MSG = 'Start context description with `when`.'
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```